src/send/saga.ts

```diff
@@ -19,14 +18,12 @@ import {
   Actions,
   HandleBarcodeDetectedAction,
   SendPaymentAction,
-  SendPaymentActionLegacy,
   sendPaymentFailure,
   sendPaymentSuccess,
   ShareQRCodeAction,
 } from 'src/send/actions'
 import { SentryTransactionHub } from 'src/sentry/SentryTransactionHub'
 import { SentryTransaction } from 'src/sentry/SentryTransactions'
-import { transferStableTokenLegacy } from 'src/stableToken/actions'
```

Comment:
I believe this whole src/stableToken directory can be deleted as this was the part to use it.